pkgs/tools/graphics/vkbasalt/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub
+, glslang, meson, ninja, pkg-config
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vkbasalt";
+  version = "0.3.2.1";
+
+  src = fetchFromGitHub {
+    owner = "DadSchoorse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0kpvgcfa5zkg1wbwhbgl16awfrh4x6za9b0rmlki8bn81f0ryvja";
+  };
+
+  # Currently doesn't support 32bit Vulkan games on 64bit systems
+  # See https://github.com/KhronosGroup/Vulkan-Loader/issues/155
```

Comment:
This will need to be absolute either way.

I am not familiar with Vulkan layers but I guess you would do something like add `${pkgsi686Linux.vkbasalt}/share` to `VK_LAYER_PATH` environment variable for 32-bit games..

Reply:
Using an absolute path by default would break multi lib compatibility on other systems, so I don't think the vkBasalt maintainer would accept the change.

It also wouldn't work to use `VK_LAYER_PATH` since the Vulkan loader does deduplication and will select the first JSON file it finds for the layer name. I suppose we can try appending the architecture to the name to workaround this.